canonical-websites/www.ubuntu.com
# 1611 cloud openstack

## Done

* update /cloud/openstack to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/cloud/openstack)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-openstack.png)

## Issue / Card

Fixed #1611
